homeassistant/components/watson_tts/tts.py

```diff
@@ -18,22 +18,32 @@
 # List from https://tinyurl.com/watson-tts-docs
 SUPPORTED_VOICES = [
     "ar-AR_OmarVoice",
+    "ar-MS_OmarVoice",
```

Comment:
According to the docs, `ar-AR_OmarVoice` is deprecated by `AR_OmarVoice`.


According to the december update: <https://cloud.ibm.com/docs/text-to-speech?topic=text-to-speech-release-notes#December2020>

A lot of voices will be removed and replacements added. Currently, this PR only processes the new additions. Should it clean up as well?

Reply:
I preferred not to break existing user configuration, and scoping my changes to only adding new voices. Maybe another update, with deprecation warnings, could handle the removal of old voices ?

Reply:
IMHO this belongs in the same update. As additions made here, are not additions but renames.

Reply:
Sure, but according to the documentation you previously linked:

> It will continue to function for at least one year but might be removed at a future date.

So, in my opinion, it should be easier for users to be warned about a future deprecation of the voice they are using, instead of breaking their current configuration during next HA update.

Reply:
> it should be easier for users to be warned about a future deprecation of the voice they are using

Yes, thus a deprecation warning to the code should be added when those voices are used.

Reply:
I'll do it as advised. Thanks for your input.